vendor/riscv/riscv-isa-sim/riscv/Proc.cc

```diff
@@ -18,6 +18,7 @@ st_rvfi Processor::step(size_t n, st_rvfi reference) {
   memset(&rvfi, 0, sizeof(st_rvfi));
 
   this->taken_trap = false;
+  this->which_trap = 0;
```

Comment:
This is a modification of Spike, is it not?  I thought we were _extending_ Spike, not modifying it.

Reply:
This is not a base spike file, so not a modification itself

Reply:
TODO delete `which_trap`